blt/blt.yml

```diff
@@ -90,6 +90,7 @@ multisites:
   - datasciencemajor
   - default
   - dennylab
+  - dfetter2022
```

Comment:
This should have been `dfetter2022__humsci`

Reply:
Ahh okay. Probably the same for the heidi one too.

Reply:
Do you want me to fix it?